tests/tornado/test_hyperbahn.py

```diff
@@ -44,6 +46,31 @@ def test_new_client_establishes_peers():
         assert channel.peers.lookup(router)
 
 
+def test_new_client_establishes_peers_from_file():
+
+    host_path = os.path.join(
+        os.path.dirname(os.path.realpath(__file__)),
+        '../data/hosts.json',
+    )
```

Comment:
For the tests, it's safe to assume that the current directory is the project root.